package.json

```diff
@@ -38,10 +38,10 @@
     "pretest": "npm run build && npm run lint",
     "reportCoverage": "nyc report --reporter=json && codecov -f coverage/coverage-final.json --disable=gcov",
     "smoketest": "./smoketests/smoketests.sh",
-    "test": "nyc jest --maxWorkers=4 --reporters=default --reporters=jest-junit",
-    "test:cli": "nyc jest test/ --maxWorkers=4 --reporters=default --reporters=jest-junit",
-    "test:packages": "nyc jest packages/ --maxWorkers=4 --reporters=default --reporters=jest-junit",
-    "test:ci": "nyc jest --maxWorkers=$(nproc) --reporters=default --reporters=jest-junit --colors",
+    "test": "nyc jest --maxWorkers=4 --reporters=default --reporters=jest-junit --no-colors",
+    "test:cli": "nyc jest test/ --maxWorkers=4 --reporters=default --reporters=jest-junit  --no-colors",
+    "test:packages": "nyc jest packages/ --maxWorkers=4 --reporters=default --reporters=jest-junit  --no-colors",
+    "test:ci": "nyc jest --maxWorkers=$(nproc) --reporters=default --reporters=jest-junit --no-colors",
```

Comment:
Colors are needed because `webpack-cli` has a `--color` flag that enables and disables colors in the help.

Reply:
Yeah, the CLI output is handled by reading the stdout stream, hence disabling the colour will disable colours only for the jest view part. Hence disabling it won't affect any of the tests related to colour characters.

![Screenshot from 2019-12-07 23-13-01](https://user-images.githubusercontent.com/32242596/70378532-5f45be00-1947-11ea-8d30-53f90c0e4b31.png)

Reply:
:arrow_up_small: in spite of disabling the colour, the logs can have colours.

Reply:
@pranshuchittora Thanks for insight 👍